generators/generator-bot-adaptive/README.md

```diff
@@ -1,9 +1,23 @@
 # @microsoft/generator-bot-adaptive [![NPM version][npm-image]][npm-url]
-> Yeoman generator for creating an Adaptive bot built on the Azure Bot Framework.
 
-## Installation
+> Yeoman generator for creating a completely empty bot built on the Azure Bot Framework & Composer.
 
-First, install [Yeoman](http://yeoman.io) and @microsoft/generator-bot-adaptive using [npm](https://www.npmjs.com/) (we assume you have pre-installed [node.js](https://nodejs.org/)):
+The Adaptive Bot generator is the base generator for creating an Azure Bot Framework bot using the Adaptive Dialog stack. This generator is designed to be used as a base for other more purpose-specific generators, and is responsible for scaffolding:
+
+- The basic bot project (code files, root dialog, base schema) for your bot, based on the platform you choose (dotnet or JavaScript)
+- The publishing environment for your bot (Azure Functions, or a Web App)
```

Comment:
Ok - I'll take another pass at this after build. I'm less concerned since it won't be in the Composer UI, which should limit discoverability pretty drastically.